### DISCUSSION TITLE
[Support]: change storage location?

### DISCUSSION THREAD

**Discussion by z1haze** [blakeblackshear/frigate#11146] [General Support]
### Describe the problem you are having

I have frigate installed via an lxc which is setup with the default storage location of /media/frigate, etc. I've just purchased a new drive that I've installed on my host and I've mounted a storage path in the resources configuration. I cannot find anywhere in the documentation where you can do to update/change the storage path for frigate clips/recordings, etc.

### Version

0.13.2

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

CPU (no coral)

### Network connection

Wired

### Camera make and model

N/A

**Comment by NickM-27** [comment]
this is meant to be done via the container config. Map the mounted volume to /media/frigate/recordings in the container

**Reply by z1haze** [reply]
Right, but after the lxc is created how are you to change it? Where is the this config stored? It appears that /media/frigate is the only option and that just seems strange. What if I wanted them in a folder called cctv?

**Reply by NickM-27** [reply]
You would create a volume in the container config that maps `/cctv:/media/frigate/recordings`

if you are using an official installation method (direct lxc) then you are on your own on how to make that work.

**Comment by Unspec7** [comment]
I too have Frigate on a LXC and am currently just symlinking it. I prefer the flexibility of symlinking. Assuming you've mounted your other drive correctly in Proxmox:

For a seamless move, shut down frigate:

`systemctl stop frigate` 

Move all the files in /media/frigate to wherever you want (I am using /rbulk/media/frigate, yours might differ depending on how you've mounted it. I like to mimic the original folder hierarchy, but you don't need to do that if you don't want to):

`mv /media/frigate/* /rbulk/media/frigate`

Delete the now empty /media/frigate

`rm -r /media/frigate`

Make the symlink:

`ln -s /bulk/media/frigate /media/frigate`

Verify that the symlink was made:

`ls -l /media/frigate`

Output:
[blahblahblah] /media/frigate -> /bulk/media/frigate

Restart frigate:

`systemctl start frigate`

Finally, go to System metrics > Storage and you'll see that the Recordings drive now matches your new capacity (in my case, from the 20GB allocated for Frigate on the boot SSD to 256GB on my spinning rust)

Use sudo if needed, I just use the LXC's root user since it's an unprivileged LXC so the security implications are less severe. I don't allow unprivileged root users to SSH into their respective LXC's either.

If you ever want to undo the symlink:

`unlink /media/frigate`

Verify no link:

`ls /media/frigate`

Output:
ls: cannot access '/media/frigate': No such file or directory
